src/daemon/variables_entrypoint.sh

```diff
@@ -98,6 +98,12 @@ if [[ "$KV_TYPE" == "etcd" ]]; then
   ETCDCTL_OPTS=(--peers ${ETCD_SCHEMA}${KV_IP}:${KV_PORT})
 fi
 
+# CEPH VARIABLES
+CEPH_VERSION=__ENV_[CEPH_VERSION]__
+CEPH_POINT_RELEASE=__ENV_[CEPH_POINT_RELEASE]__
+export CEPH_VERSION
```

Comment:
can't we combine this into 2 lines instead?

```
export CEPH_VERSION=__ENV_[CEPH_VERSION]__
export CEPH_POINT_RELEASE=__ENV_[CEPH_POINT_RELEASE]__
```

Reply:
That is triggering SC issues ;)

Reply:
Mouarf ok.

Reply:
I don't feel great about having these template variables in non-build (i.e., Dockerfile) files.  `CEPH_VERSION` is already set in the dockerfile by this line `ENV CEPH_VERSION __ENV_[CEPH_VERSION]__`. I think we should just do the same for `CEPH_POINT_RELEASE`.

Reply:
Aside:
In retrospect, allowing the templating system to do replacements in all source files was, IMO, a bad idea. I think it safer to only template the build files (we only have the Dockerfile at present, but we may have others if we support other container technologies in the future).

Reply:
Update: I wanted to make sure that `ENV` variables were available at container runtime, and it seems that this is correct. TL;DR: `ENV` is available during build and during container runtime, and `ARG` is available only at build.

https://vsupalov.com/docker-arg-env-variable-guide/#arg-and-env-availability